gravitee-am-ui/src/app/domain/applications/application/advanced/factors/step-up-auth/mfa-step-up.component.ts

```diff
@@ -45,4 +47,23 @@ export class MfaStepUpComponent {
       },
     });
   }
+
+  onToggle($event: any): void {
+    this.active = $event.checked;
```

Comment:
you can use active from stepUpRule instead of intraducting new @Input() active: boolean;